worlds/mmbn3/__init__.py

```diff
@@ -15,6 +15,7 @@
 from .Regions import regions, RegionName
 from .Names.ItemName import ItemName
 from .Names.LocationName import LocationName
+from ..generic.Rules import add_item_rule
```

Comment:
Change this to a absolute import. If you have any other relative imports, I'd recommend changing those too.

```suggestion
from worlds.generic.Rules import add_item_rule
```